src/components/ContentWidgets/ImagesInRow/ImagesInRow.css

```diff
@@ -29,5 +33,6 @@ img {
 }
 
 figcaption {
```

Comment:
Try to avoid doing this for the entire element. It will override all the `figcaption` styles because our CSS compiles to a single file. Use a class name instead, unique to your specific component.